examples/query-log-example-2c.json

```diff
@@ -0,0 +1,17 @@
+{
+    "version": "2c”,
```

Comment:
You've got some weird characters on here.  Some of the closing " aren't actually "s.